terraform/openstack/README.md

```diff
@@ -0,0 +1,95 @@
+# Work in progress!
+  Dont expect this to work yet!
```

Comment:
Shouldn't you be doing some real work ;).
Actually, do appreciate that, I need to take it out of the readme, missed it.